driver/syscall_table32.c

```diff
@@ -0,0 +1,20 @@
+/*
+
+Copyright (C) 2023 The Falco Authors.
+
+This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+or GPL2.txt for full copies of the license.
+
+*/
+
+#include <linux/kconfig.h>
+
+#if defined(__KERNEL__) && defined(CONFIG_IA32_EMULATION)
+
+#include <asm/unistd_32.h>
+
+#define g_syscall_table g_syscall_ia32_table
```

Comment:
I might be still off so forgive the possibly stupid question; shouldn't this be 
```
#define g_syscall_ia32_table g_syscall_table
```
?

Reply:
Nope, I'm pretty sure it's OK.

we `#define g_syscall_table g_syscall_ia32_table` and then `#include "syscall_table.c"`, which has:

```
const struct syscall_evt_pair g_syscall_table[SYSCALL_TABLE_SIZE] = {
```

so with the above #define it will actually create a `g_syscall_ia32_table` array

Reply:
I see now :) thank you very much! (needed a :coffee: too :laughing: )